### ISSUE TITLE
Consumer group id creation (via Kafka indexer)

### ISSUE THREAD

**Issue by abk-code** [apache/druid#8582]
Please provide a detailed title (e.g. "Broker crashes when using TopN query with Bound filter" instead of just "Broker crashes").

### Affected Version
15.1
The Druid version where the problem was encountered.

### Description

Please include as much detailed information about the problem as possible.
- 5 node 
- Configurations in use
- Steps to reproduce the problem

I'm doing some POC where I am using Kafka connector/indexer service to ingest the data into Druid.We are going to have multiple topics and consumers will be Druid (using indexer service). Ingestion works ok so far. But I am wondering what is the way to create SINGLE topic and let multiple consumers can consume using group_id.  Idea is to have different rollups (materialized views) using different consumer on same kafka topic. 

In the specfile I specify something like following, but it's not really creating consumer group when
I verify the same on Kafka side. Please help.

Here is the excerpt for the specfile on the Druid side.
Basically this is part of ioConfig > consumerProperties.
"ioConfig:" {
...
"consumerProperties": {
"group.id" : "group_id" 

}

IF there is a template to use the right specfile format, please share. 

- The error message or stack traces encountered. Providing more context, such as nearby log messages or even entire logs, can be helpful.
No error perse, since there are no CG created from the connector/indexer.

- Any debugging that you have already done
Verified that CG are not created on Kafka side by listing the consumer group.
Created multiple topics and let the consumer run on multiple topics for data ingestion.

**Comment by vogievetsky** [comment]
You can create multiple data sources on the same topic each with a different filter. Would that work?

**Comment by abk-code** [comment]
Can you please elaborate how? Do you have an example spec file? Thanks

**Comment by vogievetsky** [comment]
If you are using the streaming data loader (added in 0.16.0) then in this step:

![image](https://user-images.githubusercontent.com/177816/65921160-4e505d80-e396-11e9-837e-8ea1baae4cdd.png)

You can enter a filter to only look at a specific value

![image](https://user-images.githubusercontent.com/177816/65921200-688a3b80-e396-11e9-848b-2d29d8268ac7.png)

The spec from ☝️ is:

```json
{
  "type": "kafka",
  "ioConfig": {
    "type": "kafka",
    "consumerProperties": {
      "bootstrap.servers": "172.31.15.10:9092"
    },
    "topic": "wikiticker"
  },
  "tuningConfig": {
    "type": "kafka"
  },
  "dataSchema": {
    "dataSource": "wikiticker",
    "granularitySpec": {
      "type": "uniform",
      "queryGranularity": "HOUR",
      "segmentGranularity": "HOUR",
      "rollup": true
    },
    "parser": {
      "type": "string",
      "parseSpec": {
        "format": "json",
        "timestampSpec": {
          "column": "timestamp",
          "format": "iso"
        },
        "dimensionsSpec": {
          "dimensions": [
            "channel",
            "cityName",
            "comment",
            "countryIsoCode",
            "countryName",
            "diffUrl",
            "flags",
            "isAnonymous",
            "isMinor",
            "isNew",
            "isRobot",
            "isUnpatrolled",
            "namespace",
            "page",
            "regionIsoCode",
            "regionName",
            "user"
          ]
        }
      }
    },
    "metricsSpec": [
      {
        "name": "count",
        "type": "count"
      },
      {
        "name": "sum_added",
        "type": "longSum",
        "fieldName": "added"
      },
      {
        "name": "sum_commentLength",
        "type": "longSum",
        "fieldName": "commentLength"
      },
      {
        "name": "sum_deleted",
        "type": "longSum",
        "fieldName": "deleted"
      },
      {
        "name": "sum_delta",
        "type": "longSum",
        "fieldName": "delta"
      },
      {
        "name": "sum_deltaBucket",
        "type": "longSum",
        "fieldName": "deltaBucket"
      },
      {
        "name": "sum_metroCode",
        "type": "longSum",
        "fieldName": "metroCode"
      }
    ],
    "transformSpec": {
      "filter": {
        "type": "selector",
        "dimension": "channel",
        "value": "#es.wikipedia"
      }
    }
  }
}
```

Notice the `transformSpec.fitler`
